src/operator/quantization/quantize_graph_pass.cc

```diff
@@ -159,7 +159,7 @@ Graph QuantizeGraph(Graph &&src) {
         uint32_t min_index = 1;
         uint32_t max_index = 2;
         if (quantized_op_map.count(e.node->op())) {
-          size_t  num_outputs = e.node->num_outputs();
+          size_t  num_outputs = mirror_node->num_outputs() - 2;
```

Comment:
This should be reverted, we used a separate pass function for CPU so don't need to change this function.

Reply:
did you revert the change?

Reply:
I think this change should apply for all cases, since here we want to add min/max data (output from new_node's previous node) to the quantized new_node's input, so looks it is more proper to use previous node (also quantized version node, in this case is mirror_node) to calculate the number_outputs and the min/max index (it's possible one OP's FP32 and INT8 version can have different output so using FP32 version to calculate may not be able to represent INT8's case, for example, for CPU, FP32 and INT8 pooling can have different output number)
@reminisce Could you help to review the logic here?

Reply:
I agree with the statement that "one OP's FP32 and INT8 version can have different output", but in `mirror_node->num_outputs() - 2`, are you assuming that there are only one min and one max values?

Reply:
Yes, currently all quantized op only have 1 min and 1 max output so I think it's ok to make the assumption here? Or is there any better suggested way to handle this?